xml/obs_best_practice_staging_workflow.xml

```diff
@@ -202,5 +202,43 @@
         </para>
       </listitem>
     </itemizedlist>
+    <sect3>
+      <title>Create Staging Project from Scratch</title>
+      <para>
+        Right after the creation of a Staging, two new staging projects are
+        automatically created and assigned to it: Staging:A and Staging:B. However,
+        it is also possible to create a new staging project from scratch.
+
+        Being placed on the Staging dashboard, click on
```

Comment:
[nitpick] Add an ending para tag and and starting para tags?

`</para><para>`

Reply:
sure!